examples/tree_2d_dgsem/elixir_linearized_euler_convergence.jl

```diff
@@ -0,0 +1,61 @@
+using OrdinaryDiffEq
```

Comment:
For consistency, I think the elixirs should be named `elixir_linearizedeuler_xxx.jl`